docs/tutorials/multimodal/object_detection/detection_eval_ddetr_coco.md

```diff
@@ -0,0 +1,92 @@
+# AutoMM Detection - Evaluate Pretrained Deformable DETR on COCO Format Dataset
+:label:`sec_automm_detection_eval_ddetr_coco`
+
+In this section, our goal is to evaluate Deformable DETR model on COCO17 dataset in COCO format. 
+Previously we introduced 2 classic models: :ref:`sec_automm_detection_eval_yolov3_coco` and :ref:`sec_automm_detection_eval_fasterrcnn_coco`.
+Recent years Transformer models become more and more popular in Computer Vision, and Deformable DEtection TRansformer (Deformable DETR) reached the SOTA performance in detection task.
+In terms of speed, it's slower than YOLOv3 and Faster-RCNN, but it also has higher performance.
+
+To start, import MultiModalPredictor:
+
+```python
+from autogluon.multimodal import MultiModalPredictor
+```
+
+We select the two-stage Deformable DETR with ResNet50 as backbone with bounding box finetune,
+for other Deformable DETR models see [MMDetection's Deformable DETR models](https://github.com/open-mmlab/mmdetection/tree/master/configs/deformable_detr). 
+And we use all the GPUs (if any):
+
+```python
+checkpoint_name = "deformable_detr_twostage_refine_r50_16x2_50e_coco"
+num_gpus = -1  # use all GPUs
+```
+
+We create the MultiModalPredictor with selected checkpoint name and number of GPUs.
+We also need to specify the problem_type is `"object_detection"`.
+
+```python
+predictor = MultiModalPredictor(
+    hyperparameters={
+        "model.mmdet_image.checkpoint_name": checkpoint_name,
+        "env.num_gpus": num_gpus,
```

Comment:
it seems that we can only use one gpu in notebooks

Reply:
Yes, and we don't want the detection's tutorials run here. We directly provide the outputs.